http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/api/RuntimeRoutes.scala

```diff
@@ -378,7 +378,16 @@ object RuntimeRoutes {
       as <- x.downField("allowStop").as[Option[Boolean]]
       ap <- x.downField("autopause").as[Option[Boolean]]
       at <- x.downField("autopauseThreshold").as[Option[Int]]
-    } yield UpdateRuntimeRequest(rc, as.getOrElse(false), ap, at.map(_.minutes))
+      ul <- x.downField("labelsToUpsert").as[Option[LabelMap]]
+      dl <- x.downField("labelsToDelete").as[Option[List[String]]]
+    } yield {
+      val labelsToDelete =
```

Comment:
Could this be written as:
```
val labelsToDelete = dl.getOrElse(Set.empty).removedAll(defaultLabelKeys.defaultLabels.keySet)
val labelsToUpdate = ul.getOrElse(Map.empty).removedAll(defaultLabelKeys.defaultLabels.keySet).filter(_._2.nonEmpty)
```

Reply:
Actually -- I'm thinking it might be better to return `BadRequest` with an error message to the caller if they passed any invalid labels. The caller is probably doing something wrong in this case -- better to fail explicitly than silently ignoring the bad data.

Reply:
Please see the latest commit - I have updated so that the decoder will now return a `Error: Bad Request` rather than filtering, as you suggested